sdk-extensions/aws/src/test/java/io/opentelemetry/sdk/extension/aws/resource/LambdaResourceTest.java

```diff
@@ -41,13 +41,14 @@ void shouldAddNonEmptyAttributes() {
     Attributes attributes = resource.create().getAttributes();
 
     // then
-    assertThat(attributes)
-        .isEqualTo(
+    assertThat(attributes.asMap())
```

Comment:
Do we need to make these changes? Since we're creating the resource here I guess it doesn't have service.name etc in it.

Reply:
Funny thing...all Resources now have to have a service.name. Otherwise, how could you make sure there's always a fallback, no matter what?